android/src/main/java/community/revteltech/nfc/NfcManager.java

```diff
@@ -498,6 +527,46 @@ public void mifareUltralightWritePage(int pageOffset, ReadableArray rnArray, Cal
         }
     }
 
+    @ReactMethod
+	public void mifareUltralightWriteMultiplePage(String dataObject, Callback callback) {
+		synchronized (this) {
+			if (techRequest != null) {
+				try {
+					JSONObject jsonObject = new JSONObject(dataObject);
+					String keying = "";
+
+					Iterator<String> keys = jsonObject.keys();
+
+					while (keys.hasNext()) {
+						String key = keys.next();
+						JSONArray rnArray = (JSONArray) jsonObject.get(key);
+
+						try {
+							int pageOffset = Integer.parseInt(key);
+							byte[] bytes = jsonArrayToBytes(rnArray);
+
+							keying = keying + "|| key = " + pageOffset + ", arrayData = " + bytes.toString();
+
+							MifareUltralight techHandle = (MifareUltralight) techRequest.getTechHandle();
```

Comment:
You can define `techHandle` into upper `try` `catch` block (line 534)

Reply:
ok will try to do that.